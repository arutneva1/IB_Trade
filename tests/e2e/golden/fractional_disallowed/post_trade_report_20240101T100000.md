| symbol | side | filled_shares | avg_price | notional | avg_slippage | residual_drift_bps |
| --- | --- | --- | --- | --- | --- | --- |
| AAA | BUY | 1001.0000 | 100.50 | 100600.50 | 0.00 | -9.50 |
