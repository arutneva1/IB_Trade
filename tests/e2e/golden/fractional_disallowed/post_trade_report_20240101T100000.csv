symbol,side,filled_shares,avg_price,notional,avg_slippage,residual_drift_bps
AAA,BUY,1001.0,100.5,100600.5,0.0,-9.5
